app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="nav_bar">
     <%= link_to "Sign out", destroy_user_session_path %>
     <%= link_to "Home", root_path%>
-    <%= link_to "Posts", posts_path %>
+    <%= link_to "All Posts", posts_path %>
```

Comment:
makes sense now! it was kinda silly before when they were just two views of the same thing!